### ISSUE TITLE
POS sample fails when Nuget reference to Microsoft.Azure.Cosmos is added to CommerceRuntime sample

### ISSUE THREAD

**Issue by MaxWilson** [microsoft/Dynamics365Commerce.ScaleUnit#11]
[Possibly related to [#8](https://github.com/microsoft/Dynamics365Commerce.ScaleUnit/issues/8)]

Consuming Microsoft.Azure.Cosmos in sample CommerceRuntime.csproj leads to build-time errors in sample POS.csproj.

# Minimal repro:

1.) Clone the repo.
2.) Check out branch 9.34. (Does not repro on 9.32.)
3.) In src\ScaleUnitSample\CommerceRuntime\CommerceRuntime.csproj, add a reference to Microsoft.Azure.Cosmos. I don't think the version matters but I used 
```
<PackageReference Include="Microsoft.Azure.Cosmos" Version="3.23.0" />
```
4.) Add a reference somewhere to a type from Microsoft.Azure.Cosmos, e.g. I added to BoundController.cs 
```
    public class BoundController : IController
    {
        private Microsoft.Azure.Cosmos.CosmosClient stub = null; // add a reference to a type from Microsoft.Azure.Cosmos 
                                                                    // so the assembly won't be ignored by the compiler
        ...
```
5.) Build the solution.

**EXPECTED:** everything builds correctly, ScaleUnit package is created.
**ACTUAL:** strange error from TypeScript compilation from POS 

src\ScaleUnitSample\POS\Controls\Dialogs\Create\ExampleCreateDialogModule.ts

```
Build:Property 'EntityName' is missing in type '{ UnusualEntityId: number; IntData: number; StringData: string; ExtensionProperties: { Key: string; Value: {}; }[]; }' but required in type 'ExampleEntity'.

```

**Comment by MaxWilson** [comment]
I think this might be a regression because I did not see this issue when I filed https://github.com/microsoft/Dynamics365Commerce.ScaleUnit/issues/8

**Comment by Pikhulya** [comment]
Resolving as this was fixed on March 17, 2022 in the build 10.26.22076.1.
